app/views/posts/index.html.erb

```diff
@@ -1,7 +1,11 @@
+<h1>Welcome You are Logged In, <%= current_user.username %></h1>
```

Comment:
It might read better as "Welcome <username>" rather than "Welcome You are Logged In, <username>"